draft-yasskin-webpackage-use-cases.md

```diff
@@ -372,6 +376,30 @@ Associated requirements:
   origin.
 * {{transfer-compression}}{:format="title"}
 
+### Pre-installed applications {#preinstall}
+
+Device manufacturers would like to ship their devices with some web applications
+pre-installed and usable even if the application is first used without an
+internet connection. Thereafter, the application should use the normal Service
+Worker update mechanism to stay up to date.
+
+One way to accomplish this would be to pre-create a browser profile in the
+device's default browser, and navigate it to each of the pre-installed apps
+before recording the device image. However, this doesn't help users who change
+their default browser.
```

Comment:
it's also a pretty weird build step, isn't it?   If you built a million devices each of them would do some browsing before shipping?  and if you have actual user profiles that will be established somehow along the way, I'm not sure how that would work either.

Reply:
Yeah, maybe you can build images from the post-browsing filesystem, but now all your cookies are shared. And the browser's out-of-box experience is gone.